packages/loopring_v3/contracts/aux/liquidity-providers/BasicLiquidityProvider.sol

```diff
@@ -0,0 +1,160 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2017 Loopring Technology Limited.
+pragma solidity ^0.7.0;
+pragma experimental ABIEncoderV2;
+
+import "../agents/FastWithdrawalAgent.sol";
+import "../../lib/Claimable.sol";
+import "../../lib/EIP712.sol";
+import "../../lib/ERC20.sol";
+import "../../lib/ERC20SafeTransfer.sol";
+import "../../lib/MathUint.sol";
+import "../../lib/ReentrancyGuard.sol";
+import "../../lib/SignatureUtil.sol";
+
+
+/// @title Basic contract storing funds for a liquidity provider.
+/// @author Brecht Devos - <brecht@loopring.org>
+contract BasicLiquidityProvider is ReentrancyGuard, Claimable
```

Comment:
I suggest we use `OwnerManagable` so we can add more addresses for signing.

Reply:
Depends I guess on what we will use. More addresses that can authorize transactions (less secure I guess) vs needing multiple signatures to authorize a single withdrawal (owner could be smart wallet / multi sig contract).

Reply:
Allowing more addresses will be more flexible, for example, if we really want to speed up an transaction, we don't have to wait for all existing pending transactions, we can simply use a different address.